source/scenes/Josh/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import Logo from 'assets/Logo.svg';
+
+import Footer from 'components/Footer';
+
+import './styles.scss';
+
+const Josh = () => (
+  <div className="error">
+    <div className="content-wrapper">
+      <div className="contents">
+        <img src={Logo} alt="HackIllinois Logo" />
+        <h1>{Josh}</h1>
```

Comment:
You don't want this in curly braces. Anything in curly braces gets evaluated in React, so the h1 tag will try to display the Josh variable, which in this case is the React component itself!

To fix this, just take out the curly braces and make sure you see your name when you go to the website.